### PR TITLE
Add Фугапедия presence

### PR DESCRIPTION
**Pull request by Syjalo**
The presence for the [Fugapedia](https://fugapedia.xyz)
preview1
preview2
preview3

### CODE REVIEW

File: websites/#/Фугапедия/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
```
Comment (EncryptedDev) [inline websites/#/Фугапедия/dist/metadata.json:2]: Outdated Schema 
```suggestion
  "$schema": "https://schemas.premid.app/metadata/1.5",
```

File: websites/#/Фугапедия/presence.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const presence = new Presence({
+    clientId: "917868456232230932"
+  }),
+  startTimestamp = Math.floor(Date.now() / 1000);
```
Comment (EncryptedDev) [inline websites/#/Фугапедия/presence.ts:4]:
```suggestion
  browsingTimestamp = Math.floor(Date.now() / 1000);
```

File: websites/#/Фугапедия/presence.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+presence.on("UpdateData", async () => {
+  let presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp
```
Comment (EncryptedDev) [inline websites/#/Фугапедия/presence.ts:9]:
```suggestion
    startTimestamp: browsingTimestamp
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/#/Фугапедия/presence.ts [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: browsingTimestamp
+  }
+
+  const path = document.location.pathname;
+  if (path === "/") {
+    presenceData.details = "На главной странице";
+  }
```
Comment (Slowlife01) [inline websites/#/Фугапедия/presence.ts:15]: Remove curly brackets

File: websites/#/Фугапедия/presence.ts [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+        url: document.URL
+      }
+    ]
+  }
+  else if (path === "/search") {
+    presenceData.details = "В поисках статьи";
+  }
+  else if (path === "/create") {
+    presenceData.details = "Создаёт статью";
+  }
```
Comment (Slowlife01) [inline websites/#/Фугапедия/presence.ts:32]: This too and so on

**Review by Slowlife01 — DISMISSED** [review]

**Comment by Syjalo** [comment]
Why not add ESLint to the project?

File: websites/#/Фугапедия/dist/metadata.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    "www.fugapedia.xyz"
+  ],
+  "tags": [
+    "russian",
+    "minecraft",
+    "fugapedia"
+  ],
+  "description": {
+    "en": "Let Discord Users know what you are reading on the Fugapedia!",
+    "ru": "Дайте знать пользователям Discord о том, что вы читаете на Фугапедии!"
```
Comment (Bas950) [inline websites/#/Фугапедия/dist/metadata.json:25]: Should be a description of the website not the presence.

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**